projects/github-actions/push-to-mirrors/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - 2023-09-12
+### Fixed
+- Replaced hardcoded values of github.com with variables to support GitHub Enterprise. [#32974]
+
```

Comment:
Don't update the CHANGLOG.md files directly.

The thing to do is to create a file in `projects/github-actions/push-to-mirrors/changelog/` with appropriate contents. There are tools to help with formatting that file correctly:

* If you've done `pnpm install`, then `pnpm jetpack changelog add` should guide you through it.
* Or, in the `projects/github-actions/push-to-mirrors/` directory, you can do `composer install` and then `composer exec changelogger add`.

Reply:
My bad, I've never actually used `jetpack` so updated it manually.